crates/ruma-state-res/src/room_version.rs

```diff
@@ -74,6 +74,10 @@ pub struct RoomVersion {
     ///
     /// See: [MSC3667](https://github.com/matrix-org/matrix-spec-proposals/pull/3667) for more information.
     pub integer_power_levels: bool,
+    /// Use the `m.room.create` `sender` field as a replacement for the `creator` field.
```

Comment:
```suggestion
    /// Determine the room creator using the `m.room.create` event's `sender`,
    /// instead of the event content's deprecated `creator` field.
```

Reply:
It's not deprecated, it's removed though :stuck_out_tongue:

Reply:
Yes